service/s3/internal/s3testing/s3testing.go

```diff
@@ -0,0 +1,28 @@
+package s3testing
+
+import (
+	"fmt"
+	"math/rand"
+
+	"github.com/aws/aws-sdk-go/internal/sdkio"
+	"github.com/aws/aws-sdk-go/internal/sdkrand"
+)
+
+var randBytes = func() []byte {
+	rr := rand.New(rand.NewSource(0))
+	b := make([]byte, 10*sdkio.MebiByte)
+
+	if _, err := sdkrand.Read(rr, b); err != nil {
+		panic(fmt.Sprintf("failed to read random bytes, %v", err))
+	}
+	return b
+}()
+
+func GetTestBytes(size int) []byte {
```

Comment:
Question: Would it make sense for size to be of type int64 instead of int here?

Reply:
We would have to cast the int64 to int then for the length check then.